### PR TITLE
LuGre friction model parameters added

### PR DESCRIPTION
**Pull request by ale-git**
The LuGre friction model parameters have been added to the ergoCub robot configurations.
The template file has been updated as well.

### CODE REVIEW

**Comment by pattacini** [comment]
To avoid generating conflicts, I would give priority to merging #692 first,

File: ergoCubSN001/hardware/motorControl/left_leg-eb9-j4_5-mc.xml [hunk 1]
```diff
@@ -38,6 +38,33 @@
...
+		<param name="Kw">          0.045  0.045  </param>
+		<param name="S0">          2.5    2.5    </param>
+		<param name="S1">          0      0      </param>
+		<param name="Vth">         5      5      </param>
+		<param name="Fc_pos">      3.3    3.3    </param>
+		<param name="Fc_neg">      3.3    3.3    </param>
+		<param name="Fs_pos">      7      7      </param>
+		<param name="Fs_neg">      7      7      </param>
+	</group>
+
```
Comment (pattacini) [inline ergoCubSN001/hardware/motorControl/left_leg-eb9-j4_5-mc.xml:67]: This needs to be commented out, @ale-git

**Review by pattacini — CHANGES_REQUESTED** [review]

File: ergoCubSN001/hardware/motorControl/left_leg-eb9-j4_5-mc.xml (hunk 1 above)
Comment (ale-git) [reply ergoCubSN001/hardware/motorControl/left_leg-eb9-j4_5-mc.xml:67]: It has no effect since Km = -1, but better to remove it.

Comment (ale-git) [reply ergoCubSN001/hardware/motorControl/left_leg-eb9-j4_5-mc.xml]: I've deleted the uncommented section.

**Review by pattacini — APPROVED** [review]

File: ergoCubSN001/hardware/motorControl/left_leg-eb9-j4_5-mc.xml (hunk 1 above)
Comment (pattacini) [reply ergoCubSN001/hardware/motorControl/left_leg-eb9-j4_5-mc.xml]: Thanks!

**Review by pattacini — CHANGES_REQUESTED** [review]
As discussed T2T during the review, it'd be better off updating only the templates with the values you used during your test, @ale-git, while removing the commented sections outright.

**Comment by pattacini** [comment]
/remind December 16, @ale-git please proceed asap with this.

**Comment by ale-git** [comment]
@pattacini I've removed the LUGRE sections (apart from template).

**Comment by pattacini** [comment]
We're almost there but this PR is not really spot on yet.
Can you please give me push access to your fork?

**Review by pattacini — APPROVED** [review]
Done!

**Pull request merged**